### PR TITLE
**Feature:** Components redesign

### PR DESCRIPTION
**Pull request by fabien0102**
# Summary

image

# Related issue

# To be tested

Me
- [ ] No error or warning in the console on `localhost:6060`

Tester 1

- [ ] Things look good on the demo.
  

Tester 2

- [ ] Things look good on the demo.

### CODE REVIEW

File: src/utils/constants.ts [hunk 1]
```diff
@@ -42,8 +42,8 @@ const backgroundColors = {
   lighter: "#f2f4f6",
   /** `#f8f8f8` */
   lightest: "#f8f8f8",
-  /** `#fcfcfc` */
-  almostWhite: "#fcfcfc",
+  /** `#fafafa` */
+  almostWhite: "#fafafa",
```
Comment (TejasQ) [inline src/utils/constants.ts]:
```suggestion
  almostWhite: "#fafafaBIEN",
```

**Comment by kemal-contiamo** [comment]
> * [Accordion](https://deploy-preview-1233--operational-ui.netlify.com/#/Components/Accordion)
>   
>   * There's a 2px border when collapsed sections are adjacent to each other. Is this intentional/part of the design? Or are the borders meant to collapse and maintain 1px between sections? (@kemal-contiamo)
>     
>     * image
>   * The hover on the action is quite dark (see below). Is this intended?
>     
>     * image
> * [Tabs](https://deploy-preview-1233--operational-ui.netlify.com/#/Components/Tabs)
>   
>   * The blue tab looks quite odd contrast-wise next to and atop the greys. Can/should we adjust this?
>   * The hover of icons is also quite dark? Should we lighten it?
>     
>     * image

@TejasQ:
- **Accordion:** we need to maintain 1px line when we collapsed the section there, so not 2 px.
- **Hover:** good point, I would suggest to change the hover color in both icon cases to #D5D5D5

**Comment by fabien0102** [comment]
@kemal-contiamo I propose `#d9dfe3` instead of `#d5d5d5` (so we are on a theme color)

image

**Comment by kemal-contiamo** [comment]
> @kemal-contiamo I propose `#d9dfe3` instead of `#d5d5d5` (so we are on a theme color)
> 
> image

@fabien0102 does it also work for the icon in the tab? how does it look on them?

**Comment by fabien0102** [comment]
> > @kemal-contiamo I propose `#d9dfe3` instead of `#d5d5d5` (so we are on a theme color)
> > image
> 
> @fabien0102 does it also work for the icon in the tab? how does it look on them?

Probably ^^

**Comment by kemal-contiamo** [comment]
@fabien0102 if its not working there, more work on you again ;)

**Comment by TejasQ** [comment]
🤠

**Pull request merged**